worker/games.py

```diff
@@ -70,6 +73,30 @@ def is_64bit():
 MAKE_CMD = "make COMP=mingw " if IS_WINDOWS else "make COMP=gcc "
 
 
+def log(s):
+    logfile = os.path.join(os.path.dirname(os.path.realpath(__file__)), LOGFILE)
+    with LOG_LOCK:
+        with open(logfile, "a") as f:
+            f.write("{} : {}\n".format(datetime.datetime.utcnow(), s))
+
+
+def backup_log():
+    try:
+        logfile = os.path.join(os.path.dirname(os.path.realpath(__file__)), LOGFILE)
+        logfile_previous = logfile + ".previous"
+        if os.path.exists(logfile):
+            print("Moving logfile {} to {}".format(logfile, logfile_previous))
+            with LOG_LOCK:
+                os.rename(logfile, logfile_previous)
```

Comment:
This fail on Windows when "api.log.previous" already exists (it seems in WSL2 too in my tests).
https://docs.python.org/3/library/os.html
```
os.rename(src, dst, *, src_dir_fd=None, dst_dir_fd=None)
Rename the file or directory src to dst. If dst exists, the operation will fail with an OSError subclass in a number of cases:

On Windows, if dst exists a FileExistsError is always raised.
```

Reply:
Hmm. I replaced `os.rename()` by `os.replace()`. That should do the trick.